### PR TITLE
Uwa 462/sim authorizations

### PR DESCRIPTION
**Pull request by bmoelans**
https://vikingco.atlassian.net/browse/UWA-462

### CODE REVIEW

File: definitions/authorizations/authorization.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    type: string
+  state:
+    type: string
+    enum:
+    - requested
+    - cancelled
+    - confirmed
+    - denied
+    - removed
+  owner_email:
```
Comment (freeduerinckx) [inline definitions/authorizations/authorization.yaml]: Shouldn't we use the customer code?

File: definitions/authorizations/authorization.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    type: string
+  authorized_email:
```
Comment (freeduerinckx) [inline definitions/authorizations/authorization.yaml]: Shouldn't we use the customer code?

File: definitions/authorizations/authorization.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+type: object
+title: Authorization
```
Comment (freeduerinckx) [inline definitions/authorizations/authorization.yaml]: Add some more documentation via descriptions; in general and to the parameters.

File: definitions/authorizations/authorization.yaml (hunk 1 above)
Comment (bmoelans) [reply definitions/authorizations/authorization.yaml]: no, will add comment, you allow authentication by email, so considering GDPR if you return e-mail, it means you can abuse this to discover customer_id's ...

**Comment by bmoelans** [comment]
@freeduerinckx I added documentation, can you re-review.
Once merged I can validate system tests to model.

File: definitions/authorizations/authorization.yaml (hunk 3 above)
Comment (freeduerinckx) [inline definitions/authorizations/authorization.yaml]: I would call it Authorization details

File: paths/authorizations/create_and_list.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+          $ref: '#/definitions/Authorization'
+    401:
+      description: Unauthorized
+      schema:
+        $ref: '#/definitions/Status'
+
+post:
+  tags:
+    - Authorizations
+  summary: Create authorization
```
Comment (freeduerinckx) [inline paths/authorizations/create_and_list.yaml]: Create Authorization request

File: paths/authorizations/details.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
+get:
+  tags:
+    - Authorizations
+  summary: Get authorization
```
Comment (freeduerinckx) [inline paths/authorizations/details.yaml]: Get authorization details

**Comment by bmoelans** [comment]
@freeduerinckx pushed fixups

**Pull request merged**